frontend/src/components/core/StreamlitDialog/styled-components.ts

```diff
@@ -27,3 +27,9 @@ export const StyledShortcutLabel = styled.span(({ theme }) => ({
     textDecoration: "underline",
   },
 }))
+
+export const StyledDeployErrorContent = styled.div(({ theme }) => ({
+  "& > ul": {
+    paddingLeft: "1rem",
+  },
+}))
```

Comment:
Love seeing this used. I wonder if we should just increase indentation on `ul`s but not a big problem right now.

Reply:
Yeah! , you did a great job with these styled-components Ken, I'll add a bit more space.

Reply:
done